cmd/relay-gossip/main.go

```diff
@@ -121,7 +117,7 @@ var runCmd = &cli.Command{
 			fmt.Printf("%s/p2p/%s\n", a, h.ID())
 		}
 
-		t, err := ps.Join(lp2p.PubSubTopic(cctx.String("network-name")))
+		t, err := ps.Join(lp2p.PubSubTopic("drandGroup"))
```

Comment:
In the client code, you used the group hash for the topic. Shouldn't it be used here as well?